ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAPP-571 Allow means application through validator

## Description of change
Allow means application through validator 

Now that we have a route through the application that does not run a DWP check, the end submission validator needs to allow through applications that do not have a means passport - this PR adds a light touch to check that the means journey was started 

## Link to relevant ticket
We will in time revisit the way this validator works and possibly the need for it at all, once a variety of different routes through would result in a valid application. 

## How to manually test the feature
Create an application, clicking 'My client receives none of these passporting benefits', then go through the means journey, and case details steps. ensure that the application is allowed to be submitted.
